src/App.tsx

```diff
@@ -152,6 +155,17 @@ function App(): ReactElement {
     return urlUtils.loadParamsFromUrl();
   });
 
+  // Load URL parameters into the state that don't require a dataset to be loaded.
+  // This reduces flicker on initial load.
```

Comment:
reduces? 😄 honestly, I haven't really seen any flickering that was noticeable

Reply:
It'll sometimes jump between a few temporary states on load! You'll see the canvas go grey and them sometimes jump from `t=0` to whatever time is in the URL.

Reply:
ah, I rarely am loading urls with nontrivial settings in them